funnel/transports/sms.py

```diff
@@ -9,10 +15,60 @@
     TransportTransactionError,
 )
 
-__all__ = ['send']
+__all__ = ['send', 'TwilioSmsResponse']
+
+
+@dataclass_json
+@dataclass
+class TwilioSmsResponse:
+    """
+    SMS Status Message sent by Twilio for processing. See this for more information
+    https://www.twilio.com/docs/sms/send-messages
+
+    * sms_sid: Response ID of the SMS.
+    * status:  These could be any of queued, failed, sent, delivered, undelivered
+    * msg_status: Same values as status
+    * to: Destination Phone number
+    * msg_sid: Same values as sms_sid
+    * account_sid: Sending Account ID
+    * sender: Sending Phone number
+    * api_version: API Version
+    """
+
+    sms_sid: str = field(metadata=config(field_name='SmsSid'))
+    status: str = field(metadata=config(field_name='SmsStatus'))
+    msg_status: Optional[str] = field(metadata=config(field_name='MessageStatus'))
+    to: str = field(metadata=config(field_name='To'))
+    msg_sid: Optional[str] = field(metadata=config(field_name='MessageSid'))
+    account_sid: str = field(metadata=config(field_name='AccountSid'))
+    sender: str = field(metadata=config(field_name='From'))
+    api_version: Optional[str] = field(metadata=config(field_name='ApiVersion'))
+
+
+def send(phone_number: str, message: str, callback: bool = True) -> str:
+    """
+    Send the message to a given phone number based on internal rules.
+
+    :param phone_number: Phone Number
+    :param message: Message to deliver to Phone number.
+    :param callback: URL Callback for status updates
+    :return: SMS Message ID
+    """
+    if phone_number.startswith('+91'):
+        return _route_via_exotel(phone_number, message)
+    elif phone_number.startswith('+'):
+        return _route_via_twilio(phone_number, message, callback)
+    raise TransportRecipientError("No service provider available for this recipient")
 
 
-def send_via_exotel(phone_number, message):
+def _route_via_exotel(phone_number: str, message: str) -> str:
```

Comment:
It's okay to expose these for external use, in case a specific carrier is desired. The older names `send_via_exotel` and `send_via_twilio` work for that.